### ISSUE TITLE
runtime error when initializing JVM

### ISSUE THREAD

**Issue by juliendehos** [facebookincubator/Polygames#17]
## Steps to reproduce

  1. build Polygames (branch `ludiitest`) & install `Ludii.jar`
  2. run `./build/test_state`

## Observed Results

```
...
testing: Ludii Tic-Tac-Toe
intializing JVM
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f098403846d, pid=22505, tid=0x00007f097b39da00
#
# JRE version: OpenJDK Runtime Environment (8.0_222) (build 1.8.0_222-ga)
# Java VM: OpenJDK 64-Bit Server VM (25.222-bga mixed mode linux-amd64 compressed oops)
# Problematic frame:
# V  [libjvm.so+0x63f46d]  get_method_id(JNIEnv_*, _jclass*, char const*, char const*, bool, Thread*) [clone .constprop.116]+0x7d
#
# Core dump written. Default location: /home/jd/depots/github/facebookincubator/Polygames/core or core.22505
#
# An error report file with more information is saved as:
# /home/jd/depots/github/facebookincubator/Polygames/hs_err_pid22505.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#
Aborted (core dumped)
```

Call stack (in `hs_err_pid22505.log`):

```
Stack: [0x00007fff5873e000,0x00007fff5883e000],  sp=0x00007fff5882beb0,  free space=951k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
V  [libjvm.so+0x63f46d]  get_method_id(JNIEnv_*, _jclass*, char const*, char const*, bool, Thread*) [clone .constprop.116]+0x7d
V  [libjvm.so+0x63f81a]  jni_GetMethodID+0x7a
C  [test_state+0xbf309]  Ludii::LudiiGameWrapper::LudiiGameWrapper(JNIEnv_*, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)+0x59
C  0x0000000002b381e0
```

## Relevant Code

It seems that the error occurs here (`games/ludii/ludii_game_wrapper.cc:27`):

  ```cpp
	jmethodID ludiiGameWrapperConstructor =
			jenv->GetMethodID(ludiiGameWrapperClass, "<init>", "(Ljava/lang/String;)V");
  ```

**Comment by DennisSoemers** [comment]
Just to be sure, exactly which Ludii.jar file did you use?

**Comment by DennisSoemers** [comment]
@juliendehos 

Maybe line 24 should be changed to:

`ludiiGameWrapperClass = jenv->FindClass("player/utils/LudiiGameWrapper");`

i.e. `player/utils/` instead of just `utils/`. If that works, we'll need similar changes on lines 50 and 79 of this file, and lines 129 and 152 of `ludii_state_wrapper.cc`.

Is this something you can easily test? Otherwise I'll try, but won't be able to today.

**Comment by DennisSoemers** [comment]
@juliendehos This issue, as well as a few others that popped up afterwards, should now be fixed by Pull Request #18 . Note that this will also require an updated version of `Ludii.jar`! See https://github.com/facebookincubator/Polygames/pull/18#issuecomment-611977161

**Comment by juliendehos** [comment]
fixed in #18
